pkg/__std/lib/doc.go

```diff
@@ -1,4 +1,4 @@
-//go:generate go run -tags=dev assets_generate.go
+//go:generate go run -mod=readonly -tags=dev assets_generate.go
 
-// Package lib contains the JS std library, bunbled and minified.
+// Package lib contains the JS std library, bundled and minified.
```

Comment:
This comment is no longer true, we just bundle the `std/` directory without minification.

```
Package lib contains the JS std library ie. the std/dist directory.
```